test/README.md

```diff
@@ -104,7 +104,7 @@ If you want to run tests against other `channels`, you can specify them through
 `-channels`.
 
 ```bash
-go test -v -tags=e2e -count=1 ./test/e2e -channels=InMemoryChannel,KafkaChannel
+go test -v -tags=e2e -count=1 ./test/e2e -channel=in-memory,gcp-pubsub -tag e2e
```

Comment:
I don't think we need to change the `channels` flag. Right now `-channels=InMemoryChannel,KafkaChannel` works for the master branch.

Reply:
bad merge. fixing...